docs/source/resources/define-metadata.rst

```diff
@@ -162,9 +162,12 @@ Source from svn
 
    source:
      svn_url: https://github.com/ilanschnell/bsdiff
-     svn_rev: 1.1.4
+     svn_rev: 1.1.4 # (defaults to head)
      svn_ignore_externals: True # (defaults to False)
+     svn_username: username # Optional, if set must also have svn_password
+     svn_password: password # Optional, if set must also have svn_username
 
+Specify  svn_username and svn_password if credential is required for your svn repository.
```

Comment:
Storing sensitive username/passwords as plaintext within the metadata file in the recipe needs to come with a warning. It would be preferable for this documentation entry to be re-written as an optional way to source from svn using credentials; we should strongly suggest using environment variables like the following instead:
```
svn_username: {{ environ["SVN_USERNAME"] }}
svn_password: {{ environ["SVN_PASSWORD"] }}
```